### PR TITLE
Accessibility Features

### PR DESCRIPTION
**Pull request by MarcoOquendoC**
# [Portfolio: accessibility](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/6_accessability.md)

#### 2. Image Alternative Texts

- [change alt name for the hamburger icon](https://github.com/MarcoOquendoC/Portfolio/blob/9f60284577e715842c9493bee6da938290f92552/index.html)

#### 3. Text Headings - No Issues Found
- [Sub Heading Fix](https://github.com/MarcoOquendoC/Portfolio/blob/9f60284577e715842c9493bee6da938290f92552/index.html)
#### 4. Color contrast

- [increase font-weight for primary buttons](https://github.com/MarcoOquendoC/Portfolio/blob/9f60284577e715842c9493bee6da938290f92552/styles.css)
- [increase font size on tags](https://github.com/MarcoOquendoC/Portfolio/blob/9f60284577e715842c9493bee6da938290f92552/styles.css)
- [Fixed buttons for better contrast](https://github.com/MarcoOquendoC/Portfolio/blob/9f60284577e715842c9493bee6da938290f92552/styles.css)

#### 6. Interactions - No Issues Found
- [Fixed input label for better readability and Accessibility](https://github.com/MarcoOquendoC/Portfolio/blob/217b23cfeb75f04bea78ed18edebe80d135a348a/index.html)
- 
#### 9. Structure  - No issues Found

giphy

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -225,10 +225,10 @@ <h2>Skills</h2>
           <li><a href="https://www.linkedin.com/in/vinicio-oquendo-4a289156/"><img src="https://marcooquendoc.github.io/Portfolio/images/linkedin.png" alt="linkedin"></a></li>
           <li><a href="#"><img src="https://marcooquendoc.github.io/Portfolio/images/medium.png" alt="medium"></a></li>
           <li><a href="https://github.com/MarcoOquendoC"><img src="https://marcooquendoc.github.io/Portfolio/images/github.png" alt="github"></a></li>
-          <li><a href="#"><img src="https://marcooquendoc.github.io/Portfolio/images/angellist.png" alt="angellist"></a></li>
+          <li><a href="#"><img src="images/angellist.png" alt="angellist"></a></li>
```
Comment (MarcoOquendoC) [inline index.html:228]: Change

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @MarcoOquendoC , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Descriptive Pull request :heavy_check_mark: 
- Good use of Gitflow :heavy_check_mark: 
- Commits are meaningful :heavy_check_mark: 
- Great UI :heavy_check_mark:
- Clean code :heavy_check_mark:  

## Required Changes ♻️

- [x] Please modify the button's contrast. You can check it with [Aditus](https://www.aditus.io/button-contrast-checker/) 
- [x] Please check your single project technologies list Contrast Errors because it is very low
- [x] Your 3 contact form labels are missing, please add them because labels are there to connect an explanatory text with a form element.

Screenshot from 2022-07-12 21-53-08

Screenshot from 2022-07-12 21-49-09

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- I would like to recommend you include this extension in your browser's tab. You can follow [this link](https://stackoverflow.com/questions/4888377/how-to-add-a-browser-tab-icon-favicon-for-a-website).


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by algerina — APPROVED** [review]
Hi @MarcoOquendoC , 
Good job so far!

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSEs
 

 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by MarcoOquendoC** [comment]
## Thank you @codecaiine and @algerina.
Keep making microverse a great place to learn coding.

giphy

**Pull request merged**